eng/Version.Details.xml

```diff
@@ -1,29 +1,29 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.AspNetCore.Razor.Internal.Transport" Version="6.0.2-servicing.21607.18">
-      <Uri>https://github.com/dotnet/aspnetcore</Uri>
-      <Sha>35ae49849d414a0f408829c4e47f0724aa7b3d82</Sha>
+    <Dependency Name="Microsoft.AspNetCore.Razor.Internal.Transport" Version="6.0.2-1.21609.3">
```

Comment:
Don't know 😄. Not entirely sure about the implications to be frank

Reply:
@dotnet/aspnet-build any objections here? 609 > 607 so I think it should be fine?

Reply:
Build numbers are different because razor-compiler versions differently than aspnetcore, it's fine. But @pranavkm when you get back we should look into changing the name of `Microsoft.AspNetCore.Razor.Internal.Transport` in razor-compiler so it stops conflicting.

Reply:
Thank you all!